README.md

```diff
@@ -16,3 +16,60 @@ $> cp sample.env .env
 ```
 $> npm start
 ```
+
+## Localization
+
+In this project we're using [React-Intl](https://github.com/yahoo/react-intl) to localize our application and YAML for translation.
+
+#### Localize a component or page
+
+To localize a component or page you have to always include IntlMixin in your class mixins for example:
```

Comment:
Oh, I was reading things bottom up, and didn't see this before doing the comment below.